services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from db.models import MovieSession, CinemaHall, Movie
+from datetime import datetime

```

Comment:
check order of imports - build ins come first + must be blank lines between different import types